.github/workflows/ci.yml

```diff
@@ -66,7 +71,12 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Download dependencies
+        if: matrix.php != '8.0'
         run: composer update --no-interaction --prefer-dist
 
+      - name: Download dependencies (ignore-platform-req)
+        if: matrix.php == '8.0'
+        run: composer update --no-interaction --prefer-dist --ignore-platform-req=php
```

Comment:
Do we still need to ignore platform reqs now?

Reply:
No, our dependencies supports PHP8